src/synthesized_insight/metrics/base.py

```diff
@@ -83,6 +84,81 @@ def __call__(self, sr_a: pd.Series, sr_b: pd.Series) -> Tuple[Union[int, float,
         return self._compute_test(sr_a, sr_b)
 
 
+class TwoColumnMap(_Metric):
+    """Compares columns with the same name from two given dataframes and return a DataFrame
+    with index as the column name and the columns as metric_value and metric_pval(if applicable)"""
+
+    def __init__(self, metric: Union[TwoColumnMetric, TwoColumnTest]):
+        self.metric = metric
+        self.name = f'{metric.name}_map'
+        super().__init__()
+
+    def __call__(self, df_old: pd.DataFrame, df_new: pd.DataFrame) -> Union[pd.DataFrame, None]:
+        if df_old is None or df_new is None:
+            return None
```

Comment:
I think these lines aren't needed as both dfs are defined to be pd.DataFrames

Reply:
Removed these lines